sklearn/tests/test_docstring_parameters.py

```diff
@@ -84,7 +84,8 @@ def test_docstring_parameters():
             module = importlib.import_module(name)
         classes = inspect.getmembers(module, inspect.isclass)
         # Exclude imported classes
-        classes = [cls for cls in classes if cls[1].__module__ == name]
+        classes = [cls for cls in classes
+                   if cls[1].__module__.startswith(name)]
```

Comment:
@jnothman  Here is the change. The parameterize was to make it easier for me to see all the errors at once.

I reverted the parameterize so the diff is smaller.

Reply:
Do I understand correctly that the problem was that `name` is e.g. `sklearn.cross_decomposition` while `cls[1].__module__` is `sklearn.cross_decomposition._pls`?

Reply:
and if that's the case can we just do

```py
classes = [cls for cls in classes if cls[1].__module__.startswith('sklearn')]
```
?

Reply:
Yup that is nicer. Updated PR with your suggestion.